src/data/pages/lsst-hubble-plot.json

```diff
@@ -17,6 +17,10 @@
   "widget": {
     "type": "HubblePlot",
     "source": "/data/galaxies/multiple_galaxies.json",
+    "layout": {
+      "col": "left",
+      "row": "bottom"
+    },
```

Comment:
I don't see any reason why the positioning attributes shouldn't be consistent across our main page components: images, tables, and widgets.  As I understand it you are using a competing may to define position and to build the classname that kicks off your css grid stuff.  As you've done this in two different ways now, you probably have a sense of which mode is working better for ya: whichever mode that is, please update your work so it's consistent across all currently positionable components.

Reply:
I really do like this current solution better that then CSS-Grid solution. That IE issue is going to always dampen development until they completely destroy that app and no one, and I mean, no one uses IE any longer. I'm just going to have to update the tables work that I did with CSS-Grid and implement this solution.